lib/transition.js

```diff
@@ -294,6 +308,22 @@ class Transition {
       }
       return undefined
     }
+
+    /**
+     * Check if transition prop is inside vendor specific rule
+     */
+    inVendorSpec (decl) {
+      let { parent } = decl
+
+      if (parent.type !== 'rule') {
+        return false
+      } else if (!parent.selector.includes(':-')) {
+        return false
+      } else {
+        return !this.prefixes.add.selectors
+          .find(s => parent.selector.includes(s.name))
```

Comment:
1. `.some()` will be faster (and more readable)
2. Why do we need to test prefixes from Autoprefixer. Some prefixes are non-standard and Autoprefixer doesn’t support it (for instance, `::-webkit-scrollbar`).
3. We can return `declPrefixes` from `inVendorSpec` to make code more simple.

Reply:
> Why do we need to test prefixes from Autoprefixer.

Cause we don't need to handle standard selectors like ```:read-only``` and so on. The decls inside this selectors have to prefixed with all relevant prefixes.

> Some prefixes are non-standard and Autoprefixer doesn’t support it (for instance, ::-webkit-scrollbar).

And decls inside this selector will be prefixed only with ```-webkit-``` prefix.

Reply:
> Cause we don't need to handle standard selectors like :read-only and so on. The decls inside this selectors have to prefixed with all relevant prefixes.

You can use `Browsers.prefixes()` to get all prefixes and then check them in selector:

```js
} else {
  let prefixes = Browsers.prefixes().filter(i => parent.selector.includes(':-' + i))
  return prefixes.length > 0 ? prefixes : false
}
```

Reply:
Completely forgot about ```Browsers```. Thanks!